src/en/charms-storage.md

```diff
@@ -415,6 +430,72 @@ provider currently supports a single pool configuration attribute:
 
 ### LXD (lxd)
 
+LXD-based models have access to the 'lxd' storage provider. The LXD provider
+does not currently have any specific configuration options.
+
+!!! Note:
+    To get an LXD version on either Ubuntu 14.04 LTS (Trusty) or Ubuntu 16.04
+    LTS (Xenial) that has the 'lxd' storage provider feature the
+    [LXD PPA][ppa-lxd] (
+    `sudo apt-add-repository -yu ppa:ubuntu-lxc/lxd-stable; sudo apt install -y lxd`
+    ) will be required.
```

Comment:
FWIW, I'd say the minimum is 2.15: https://insights.ubuntu.com/2017/07/12/storage-management-in-lxd-2-15/. There have been fixes since then, and there's no reason not to use 2.16, so may as well just say 2.16.